src/unit.config.ts

```diff
@@ -12,7 +12,7 @@ function webpackConfig(args: any): webpack.Configuration {
 	const outputPath = output!.path as string;
 	config.entry = () => {
 		const unit = globby
-			.sync([`${basePath}/tests/unit/**/*.ts`])
+			.sync([`${basePath}/tests/unit/**/*.ts`, `${basePath}/src/**/*.spec.{ts,tsx}`])
```

Comment:
Should the other glob also include `tsx`? I know it's not part of these changes tho!

Reply:
I can get behind that change! Done!